src/integration-test/tests/citizen/dashboard/dashboard_test.ts

```diff
@@ -1,17 +1,26 @@
 import { PartyType } from 'integration-test/data/party-type'
-import { claimAmount, createDefendant } from 'integration-test/data/test-data'
+import { claimAmount, createClaimData, createDefendant } from 'integration-test/data/test-data'
 import { ClaimSteps } from 'integration-test/tests/citizen/claim/steps/claim'
 import I = CodeceptJS.I
 import { AmountHelper } from 'integration-test/helpers/amountHelper'
+import { DefendantClaimDetails } from 'integration-test/tests/citizen/defence/pages/defendant-claim-details'
 
 const claimSteps: ClaimSteps = new ClaimSteps()
+const defendantDetails: DefendantClaimDetails = new DefendantClaimDetails()
```

Comment:
Please rename to match the purpose. Page object can be used for both claimant and defendant dashboard at this point. Name like `DashbaordClaimDetails` would reflect the purpose better.